andela/firestar-backend
# configure eslint using airbnb base [finish #167730729]

#### What does this PR do?
- A summary of what this task is about
This PR configures eslint for the project
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
 installed eslint eslint-airbnb-base eslint-plugin-import
 created the .eslintrc file and stated the rules there
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
 eslint points out code that does not adhere to the standards stated on the .eslintrc file
 create a javascript file and type in: var app = 10;
 this declaration will prompt an eslint error

#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#167730729](https://www.pivotaltracker.com/n/projects/2382004)`)

#### Screenshot
- N/A
